dptb/postprocess/unified/properties/optical_conductivity.py

```
@@ -0,0 +1,247 @@
+            # Flatten for efficiency (common prep)
+            E_flat = E_mn.flatten()
+            T_weighted_flat = (T_nm * w_batch.reshape(-1, 1, 1)).flatten()
+            
+            if method == 'jit':
+                term = accumulate_sigma_jit(E_flat, T_weighted_flat, omegas_t, eta, broadening)
+            elif method == 'vectorized':
+                term = self._accumulate_vectorized(E_flat, T_weighted_flat, omegas_t, eta, broadening)
+            elif method == 'loop':
+                 term = self._accumulate_loop(E_flat, T_weighted_flat, omegas_t, eta, broadening)
+            else:
+                raise ValueError(f"Unknown method: {method}")
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)